package-lock.json

```diff
@@ -1889,7 +1889,7 @@
         },
```

Comment:
hmm, interesting that these changed from `https` -> `http`.  

could you revert these changes by running: `git checkout origin/master package-lock.json` and then commit the result?

Reply:
Done, didn't see that one!
I think you mean `git checkout origin/develop package-lock.json`

Reply:
whoops!  yes, I did mean `develop`.  thank you!